### PR TITLE
Remove unnecessary import of logging packages

### PR DESCRIPTION
**Pull request by kbaillie**
OpenLiberty Pull Requester,

ATTENTION, READ THIS: Updated 4/11/2018 - Read and understand this completely,
then delete this entire template. If a reviewer or merger sees this template,
they should fail the review or merge.

If this code change is fixing a user-visible bug in previously released code, it MUST
have an associated issue mentioned in the PR text or description. That Issue also
MUST be labelled “release bug”

This directs automation to scrape this fix for inclusion in the next release's
list of bugs fixed.

If this code is NOT for fixing a released bug, for example new function, fixing
a bug in unreleased function, or other improvements that do not affect the
user-space, no label is needed. An issue could or could not be used based
on the developer’s discretion, but do still delete this text block.

For full details, please see this wiki page:

https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions

### CODE REVIEW

**Comment by kbaillie** [comment]
#build

**Comment by kbaillie** [comment]
#build

File: dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/internal/KernelBootstrap.java [hunk 1]
```diff
@@ -39,6 +39,7 @@
 import com.ibm.ws.kernel.boot.Launcher;
```
Comment (Channyboy) [inline dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/internal/KernelBootstrap.java]: copyright

File: dev/com.ibm.ws.kernel.boot/bnd.bnd [hunk 2]
```diff
@@ -58,8 +58,7 @@ Export-Package: com.ibm.ws.kernel.launch.service;provide:=true, \
   com.ibm.ws.kernel.provisioning;provide:=true, \
```
Comment (Channyboy) [inline dev/com.ibm.ws.kernel.boot/bnd.bnd]: copyright

File: dev/com.ibm.ws.kernel.boot.core/bnd.bnd [hunk 3]
```diff
@@ -25,8 +25,7 @@ Export-Package: \
 	com.ibm.ws.kernel.internal.classloader,\
```
Comment (Channyboy) [inline dev/com.ibm.ws.kernel.boot.core/bnd.bnd]: copyright

**Comment by kbaillie** [comment]
#build

**Comment by kbaillie** [comment]
#build

**Comment by kbaillie** [comment]
#build

**Pull request merged**